buffother.lic

```diff
@@ -0,0 +1,46 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#buffother
+=end
+custom_require.call(%w(common common-arcana events spellmonitor drinfomon common-travel))
+class Buffother
+  include DRC
```

Comment:
💡 Where is `common` used?

Reply:
It was at one point, but isn't used anymore. I'll remove it on the next commit.